packages/devtools_app/lib/src/screens/debugger/debugger_model.dart

```diff
@@ -524,29 +524,6 @@ Future<void> buildVariablesTree(
   variable.treeInitializeComplete = true;
 }
 
-// TODO(elliette): Remove once the fix for dart-lang/webdev/issues/1439
-// is landed. This works around a bug in DWDS where an error is thrown if
-// `getObject` is called with offset/count for an object that has no length.
-Future<Obj> _getObjectWithRetry(
-  String objectId,
-  DartObjectNode variable,
-) async {
-  final variableId = variable.ref!.isolateRef!.id!;
-  try {
-    final dynamic result = await serviceManager.service!.getObject(
-      variableId,
-      objectId,
-      offset: variable.offset,
-      count: variable.childCount,
```

Comment:
@elliette correct me if I am wrong, but we still want `offset` and `count` to be included in the call above don't we?

Reply:
Yes, we still want to include offset and count :)